src/main/java/baritone/process/MineProcess.java

```diff
@@ -421,7 +421,7 @@ private void addNearby() {
     }
 
     public static boolean plausibleToBreak(CalculationContext ctx, BlockPos pos) {
-        if (MovementHelper.getMiningDurationTicks(ctx, pos.getX(), pos.getY(), pos.getZ(), ctx.bsi.get0(pos), true) >= COST_INF) {
+        if (MovementHelper.avoidBreaking(ctx.bsi, pos.getX(), pos.getY(), pos.getZ(),  ctx.bsi.get0(pos))) {
```

Comment:
Why change this?

Reply:
See the prior review convo where @wagyourtail explained why.

Reply:
I did not see any such explanation? `that whole function is only used for pathing goals and removing that allows pathing to "unbreakable" blocks ie. end portal frame.` This isn't a reason. If the mining duration is above the maximum cost, it is not plausible to break. What bug does this fix / why make this change?

Reply:
#goto end_portal_frame

Reply:
Your PR also makes end portal frame "can walk through" which is incorrect. That could cause untold issues and stuck paths.

Reply:
canWalkThrough on closer inspection appeared to me to be a function that should be named "canWalkOn" as that appears to be what it does else why are chests and Grass_path true